.github/workflows/update-public-docs.yml

```diff
@@ -59,31 +62,90 @@ jobs:
           prerelease: false
 
       - name: Upload Release Asset
-        if: ${{ steps.check_docs.outputs.DOCS_CHANGED }}
-        id: upload-release-asset
+        if: ${{ steps.check_release_changed.outputs.DOCS_CHANGED }}
+        id: upload_release_asset
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          upload_url: ${{ steps.create-release.outputs.upload_url }}
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
           asset_path: ./new_public_docs.json
           asset_name: public_docs.json
           asset_content_type: application/json
 
-      - name: Trigger update in docs repo
-        if: ${{ steps.check_docs.outputs.DOCS_CHANGED }}
+      - name: Clone docs repo
+        if: ${{ steps.check_release_changed.outputs.DOCS_CHANGED }}
+        uses: actions/checkout@v3
+        with:
+          repository: opensafely/documentation
+          path: docs
+          ref: 'main'
+          token: ${{ secrets.DOCS_WRITE_TOKEN }}
```

Comment:
comment: I've just merged opensafely/documentation#832 which switches cohort-extractor to now being a submodule. Submodules aren't cloned by default with `actions/checkout@v3`.

I don't believe that will make a difference here, because we're not building the docs, but just mentioning in case.

Reply:
No, that shouldn't matter